contrib/kind.sh

```diff
@@ -822,6 +827,11 @@ set_ovn_image() {
 }
 
 build_ovn_image() {
+  local push_args=""
+  if [ "$OCI_BIN" == "podman" ]; then
+    push_args="--tls-verify=false"
```

Comment:
maybe we should add a comment to "TODO: investinate if the localregistry must be using a secure connection instead?" Does this mean docker is ok with insecure conn as the default?

Reply:
ack, whatever you said here in the comment: "docker doesn't perform tls check by default only podman does" -> hence we need to disable it for podman -> if you can add that as a comment on the script it would be good info.